### PR TITLE
prep javascript functions- Hannah Hoang

### PR DESCRIPTION
**Pull request by hannahhoanggg**
Screenshot 2023-06-24 033016

### CODE REVIEW

File: prep-javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+function getCircleCircumference(radius) {
+  return radius * 6.28318530718;
+}
+
+var getCircleCircumferenceResult = getCircleCircumference(7);
+console.log('getCircleCircumference Exercise: ', getCircleCircumferenceResult);
+
+function getFullName(firstName, lastName) {
+  return firstName + lastName;
```
Comment (BrettAlbright) [inline prep-javascript-functions/main.js]: don't forget to include the space here

File: prep-javascript-functions/main.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+var getFullNameResult = getFullName('Hannah ', 'Hoang');
+console.log('getFullName Exercise: ', getFullNameResult);
+
+function cube(number) {
+  return number * 4;
```
Comment (BrettAlbright) [inline prep-javascript-functions/main.js]: this function needs to cube any number when it is called, not just the number 2.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates are needed!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**